tools/commit_slider_tool/utils/cfg.json

```diff
@@ -0,0 +1,46 @@
+{
+    "modeMap" : {
+        "checkOutput" : "CheckOutputMode",
+        "bmPerf" : "BenchmarkAppPerformanceMode"
+    },
+    "traversalMap" : {
+        "firstFailedVersion" : "FirstFailedVersion",
+        "firstFixedVersion" : "FirstFixedVersion"
+    },
+    "commandList" : [
+        {"cmd" : "git checkout -- .", "path" : "{gitPath}"},
+        {"cmd" : "git clean -fxd", "path" : "{gitPath}", "tag" : "clean"},
+        {"cmd" : "mkdir -p build", "path" : "{gitPath}"},
+        {"cmd" : "git checkout {commit}", "catchMsg" : "error", "path" : "{gitPath}"},
+        {"cmd" : "git submodule init", "path" : "{gitPath}"},
+        {"cmd" : "git submodule update --recursive", "path" : "{buildPath}"},
+        {"cmd" : "{makeCmd}", "catchMsg" : "CMake Error", "path" : "{buildPath}"},
+        {"tag" : "preprocess"},
+        {"cmd" : "make --jobs=4", "path" : "{buildPath}"},
+        {"cmd" : "git checkout -- .", "path" : "{gitPath}"}
+    ],
+    "makeCmd" : "cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_PYTHON=OFF -DTHREADING=TBB -DENABLE_MKL_DNN=ON -DENABLE_CLDNN=OFF -DENABLE_INTEL_GNA=OFF -DENABLE_INTEL_VPU=OFF -DENABLE_INTEL_MYRIAD=OFF -DENABLE_INTEL_MYRIAD_COMMON=OFF -DENABLE_HDDL=OFF -DENABLE_MODELS=OFF -DENABLE_SAMPLES=OFF -DENABLE_TESTS=ON -DENABLE_HETERO=OFF -DENABLE_TEMPLATE=OFF -DENABLE_CPU_DEBUG_CAPS=OFF -DENABLE_DEBUG_CAPS=OFF -DENABLE_OV_CORE_BACKEND_UNIT_TESTS=OFF -DENABLE_OPENVINO_DEBUG=OFF ..",
+    "returnCmd" : "git checkout master",
+    "gitPath" : "../../",
+    "buildPath" : "../../build/",
```

Comment:
What if I use another folder for the build?

Reply:
You just set it in custom_cfg.json, according to doc